examples/monitor/main.go

```diff
@@ -0,0 +1,134 @@
+// Copyright 2023 The Ebitengine Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"image/color"
+	"log"
+
+	"golang.org/x/image/font"
+	"golang.org/x/image/font/opentype"
+
+	"github.com/hajimehoshi/ebiten/v2"
+	"github.com/hajimehoshi/ebiten/v2/examples/resources/fonts"
+	"github.com/hajimehoshi/ebiten/v2/inpututil"
+	"github.com/hajimehoshi/ebiten/v2/text"
+)
+
+var (
+	mplusNormalFont font.Face
+)
+
+func init() {
+	tt, err := opentype.Parse(fonts.MPlus1pRegular_ttf)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	const dpi = 72
+	mplusNormalFont, err = opentype.NewFace(tt, &opentype.FaceOptions{
+		Size:    24,
+		DPI:     dpi,
+		Hinting: font.HintingFull,
+	})
+	if err != nil {
+		log.Fatal(err)
+	}
+}
+
+const (
+	screenWidth  = 640
+	screenHeight = 480
+)
+
+type Game struct {
+}
+
+func (g *Game) Update() error {
+	if inpututil.IsMouseButtonJustPressed(ebiten.MouseButtonLeft) {
+		cx, cy := ebiten.CursorPosition()
+		l := text.BoundString(mplusNormalFont, "|").Dy()
+		y := l
+
+		b := text.BoundString(mplusNormalFont, "toggle fullscreen")
+		if cx >= b.Min.X && cx <= b.Max.X && cy >= b.Min.Y+y && cy <= b.Max.Y+y {
```

Comment:
I don't think we need to implement a mouse-clicking UI. This would make this program a little complicated (especially to calculate the text bounds). What about treating keys like F for fullscreening and 0-9 for selecting monitors?

Reply:
Sounds reasonable and would make the code smaller.

Reply:
I have adjusted this. I did introduce:
```golang
		for i := 0; i < len(g.monitors); i++ {
			if inpututil.IsKeyJustPressed(ebiten.KeyDigit0 + ebiten.Key(i)) {
				m := g.monitors[i]
				ebiten.SetWindowTitle(m.Name())
				ebiten.SetMonitor(m)
			}
		}
```

This is more concise than checking for each monitor/digit, but could be weird.

Reply:
Acknowledged.